src/integration/pages/mediaAssetPage/user.js

```diff
@@ -0,0 +1,49 @@
+const { within } = require('@testing-library/dom');
+
+const { amp, canonical } = global;
+
+export default ({
+  headlineText,
+  timestamp,
+  bulletedListItem,
+  relatedContentHeadline,
+}) => {
+  [amp, canonical].forEach(page => {
+    describe(`And using ${page.platform}`, () => {
+      it('I can see the headline', () => {
+        const { getByText } = within(
+          page.document.querySelector('h1[id="content"]'),
+        );
+
+        const headline = getByText(headlineText);
+        expect(headline).toBeInTheDocument();
+      });
+
+      it('I can see the timestamp', () => {
+        const { getByText } = within(page.document.querySelector('time'));
+
+        const timestampEl = getByText(timestamp);
+
+        expect(timestampEl).toBeInTheDocument();
+      });
+
+      it.skip('I can see the bulleted list item', () => {
```

Comment:
If you unskip this, you will see why it fails - might need some assistance on how to fix it.